lib/plugins/awsRemoveResources/lib/removeStack.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+const async = require('async');
+const find = require('lodash').find;
+
+module.exports = {
+  remove() {
+    this.serverless.cli.log('Removing Stack...');
+
+    const stackName = `${this.serverless.service.service}-${this.options.stage}`;
+
+    const params = {
+      StackName: stackName,
+    };
+
+    return this.CloudFormation.deleteStackPromised(params).then(() => {
+      return BbPromise.resolve(params);
```

Comment:
hmmmm I dont think that will work, the `remove()` method/promise needs to resolve the `cfData` it gets from aws so that they're passed to `monitorRemove()` ... the way you have it set will pass the `params` to `monitorRemove()`